### PR TITLE
add stackoverflow to profile

### PR DESCRIPTION
**Pull request by lukaselmer**

### CODE REVIEW

File: src/documents/ueber-uns/index.html.eco [hunk 1]
```diff
@@ -104,6 +104,15 @@ position: 4
           <% if employee.xing: %>
             <a href="https://www.xing.com/profile/<%= employee.xing %>"><i title="Zum Xing Profil" class="icon-xing-sign icon-4x"></i></a>
           <% end %>
+
+          <% if employee.stack_overflow: %>
+            <a href="https://stackoverflow.com/users/<%= employee.stack_overflow %>">
+              <span class="icon-stack icon-2x" style="vertical-align: -16%; margin-left: -4px;">
```
Comment (lukaselmer) [inline src/documents/ueber-uns/index.html.eco]: it is a bit ugly, but we will have to change it anyways when we upgrade to font awesome 4

File: src/documents/ueber-uns/index.html.eco [hunk 2]
```diff
@@ -104,6 +104,15 @@ position: 4
...
+                <i title="Zum Stack Overflow Profil" class="icon-sign-blank icon-stack-base"></i>
+                <i title="Zum Stack Overflow Profil" class="icon-stackexchange icon-light"></i>
```
Comment (sihu) [inline src/documents/ueber-uns/index.html.eco]: I couldnt test yet on the browser why this are two <i>. On purpose?

Comment (lukaselmer) [inline src/documents/ueber-uns/index.html.eco]: Yep. See https://new-compare2.renuo.ch/ueber-uns/

**Comment by lukaselmer** [comment]
:+1:

**Pull request merged**